lib/commands/publish.js

```diff
@@ -196,6 +203,28 @@ class Publish extends BaseCommand {
     }
   }
 
+  async #latestPublishedVersion (spec) {
+    try {
+      const uri = '/' + spec.escapedName
+      const packument = await npmFetch.json(uri, {
```

Comment:
`pacote.packument` and `pacote.manifest` are the proper ways to fetch packuments and manifests.